### PR TITLE
Prevent total objects deletion

### PR DESCRIPTION
**Pull request by esirK**
## Description

Instead of delete action fully deleting an object e.g a Newspaper article, we should instead mark them as deleted and hide them from users. This way, we can always retrieve them in case of accidental deletion.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
QQ @esirK , How does this impact our report code?

**Comment by esirK** [comment]
I don't think it affects it in anyway @kilemensi  I'll try to generate them just to confirm

**Comment by kilemensi** [comment]
> I don't think it affects it in anyway @kilemensi I'll try to generate them just to confirm

Waiting on this @esirK

**Comment by esirK** [comment]
> > I don't think it affects it in anyway @kilemensi I'll try to generate them just to confirm
> 
> Waiting on this @esirK

@kilemensi I did generate them and nothing was affected.

**Comment by kilemensi** [comment]
> > > I don't think it affects it in anyway @kilemensi I'll try to generate them just to confirm
> > 
> > 
> > Waiting on this @esirK
> 
> @kilemensi I did generate them and nothing was affected.

Good to hear that @esirK but just to confirm: `report` being another Django app uses the same queryset to fetch the various sheets from `forms` app i.e. _report will not fetch and process Sheets that are marked as deleted_?

**Comment by esirK** [comment]
@kilemensi  I have made the fix.

File: forms/manager.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.db import models
```
Comment (kilemensi) [inline forms/manager.py]: Why not in `modelutils.py`?

**Review by kilemensi — APPROVED** [review]
:rocket: @esirK ... LGTM.

File: forms/manager.py (hunk 1 above)
Comment (esirK) [reply forms/manager.py]: @kilemensi  wanted to have a better separation but looking at it again it makes sense to have it inside this file. I've made the changes.

**Pull request merged**